### DISCUSSION TITLE
Emission locality and imports

### DISCUSSION THREAD

**Discussion by Imaskar** [electricitymaps/electricitymaps-contrib#3195] [Q&A]
Hi! It's not really a bug and I understand why is it implemented that way. I just want to start a discussion about possible improvements.

On a screenshot below we can see that Finland imports a lot of dirty electricity from Central Russia, which is 300+ on average. Howewer, we also know that that specific region is oversupplied by Kola NPP. The actual carbon intensity there is 12. Can this locality be accounted for?

![Screenshot_20210623-225800_Firefox](https://user-images.githubusercontent.com/7257720/123167298-a6869780-d476-11eb-9cca-9b588350f5df.jpg)

**Comment by FelixDQ** [comment]
The flowtracing methodology (described in a blog post here https://www.tmrow.com/blog/flow-tracing/), assumes that exports represent an equal split of the mix in a zone. i.e. a zone can't selectively export some energy sources.

The best way to account for this would be to have more granular data sources. I.e. if we can find a data source that provides more regional data.
